app/dockerdwrapper.c

```diff
@@ -851,13 +877,25 @@ main(void)
   return exit_code;
 }
 
+static const char *
+str_rm(int method)
+{
+  switch (method) {
+    case POST:
+      return "POST";
+    case DELETE:
+      return "DELETE";
+  }
+  return "UNKNOWN";
+}
+
 void
 callback_action(__attribute__((unused)) fcgi_handle handle,
                 int request_method,
                 char *cert_name,
                 char *file_path)
 {
-  syslog(LOG_INFO,"callback_action hit with request %i and cert name %s, filepath %s", request_method, cert_name, file_path);
+  syslog(LOG_INFO,"callback_action: request %s with cert_name %s, file_path %s", str_rm(request_method), cert_name, file_path);
```

Comment:
I added this logging more as a debug during development - I would rather you remove it than add extra code to support it

Reply:
Done